_posts/2017-08-14-fec-case-study.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How the FEC.gov team uses the Web Design Standards
+tags:
+- web design standards
+- design
+- case study
+- design
+---
+The U.S. Web Design Standards are currently being used on hundreds of government
+sites, with an audience of more than 59 million monthly users. In this eighth
+post in our series, we met with Jennifer Thibault, visual designer and
+innovation specialist, to talk about the Federal Election Commission (FEC)’s use
+of the Standards on [fec.gov][].
+
+**Standards team:** How did you find out about the U.S. Web Design Standards?
+
+**Jennifer Thibault:** We knew of the USWDS project early on since it was
+kicked off by a team inside 18F. But, we needed to get out a basic set of
+pages and content models on fec.gov [before][] the Standards team released
```

Comment:
Hmm, this isn't a very accessible hyperlink because the link text isn't very useful on its own.  See the [Screen reader users often navigate from link to link, skipping the text in between](http://webaim.org/techniques/hypertext/#screen_readers) section of WebAIM's _Introduction to Links and Hypertext_ for more details.

That said, I'm not sure how best to change it, since it's hyperlinking Thibault's verbatim dialogue. One way to address it might be to have a section under the dialogue that hyperlinks to the same place with more contextual link text, but italicizes the text to indicate that it's not verbatim dialogue?  I'm not sure--we could also just leave it as-is, I guess.